pixeltable/share/publish.py

```diff
@@ -41,23 +45,23 @@ def push_replica(
 
     # Create the publish request using packager's bundle_md
     publish_request = PublishRequest(
-        table_uri=PxtUri(dest_tbl_uri),
+        table_uri=PxtUri(uri=dest_tbl_uri),
         pxt_version=packager.bundle_md['pxt_version'],
         pxt_md_version=packager.bundle_md['pxt_md_version'],
         md=packager.bundle_md['md'],
         bucket_name=bucket,
         is_public=access == 'public',
     )
 
-    response = requests.post(PIXELTABLE_API_URL, json=publish_request.model_dump_json(), headers=_api_headers())
+    response = requests.post(PIXELTABLE_API_URL, data=publish_request.model_dump_json(), headers=_api_headers())
     if response.status_code != 200:
-        raise excs.Error(f'Error publishing snapshot: {response.text}')
-    publish_response = PublishResponse.model_validate_json(response.json())
+        raise excs.Error(f'Error publishing replica: {response.text}')
```

Comment:
Shouldn't this be 'error publishing {src_tbl._display_name()}'? I thought we use replica purely to refer to local replicas of a published table.